config/initializers/omniauth.rb

```diff
@@ -12,7 +12,7 @@
            },
            name: 'CCID'
 
-  provider :developer unless Rails.env.production?
+  provider :developer unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
```

Comment:
Not to bikeshed but if it's hard to read make it a proper If conditional and not have it inline?
This cop is for protecting against this, which honestly I would agree, you don't need parenthesis
```# bad
if (x > 10)
  # body omitted
end
```
```
# good
if x > 10
  # body omitted
end
```